.travis.yml

```diff
@@ -1,10 +1,19 @@
 language: php
+
 php:
   - 7.0
   - 7.1
+  - 7.2
+
+env:
+  - COMPOSER_FLAGS=--prefer-lowest
+  - COMPOSER_FLAGS=
+
 before_script:
-  - composer install
+  - composer update --no-interaction --no-suggest $COMPOSER_FLAGS
```

Comment:
No consequence with flags. Move all to environment variable.

Reply:
`--no-interaction --no-suggest` are constant so we don't need to have it into variable